Open_Science_New.html

```diff
@@ -164,78 +203,93 @@ <h2><Demo</h2>
-	    
-	    <p><font color="yellow">Enable scientific discovery</font></p>
-	    <p><font color="yellow">It's the future!</font></p>
-	    <a href="https://www.youtube.com/watch?v=Wtp7R97jz5k" target="_blank"><p><small><font color="orange">Aled Edwards video</font></small></p></a>
+	    <a href="https://samirdas.github.io/MNI_ecosystem.html#/"><img src="images/mni_eco_info_5.jpg" data-background-size="100%"></a>
+          <p><small>View this presentation (click image to open) for further details
+              on our initiatives at the MNI.</small></p>
+	    </section>
+
+          </section>
+
+          <!-- HBHL Column 4-->
+          <section>
+              <h1>HBHL</h1>
+                      <p>Healthy Brains, Healthy Lives</p>
+              <a href="https://www.mcgill.ca/hbhl/"><img src="images/hbhl_logo.png"></a>
+              <p><small><i>Exploring our brains' role in health and disease throughout our lives.</i></small></p>
+              </section>
+
+          <!-- CONP Column 5-->
+              <section>
+                  <section>
+                      <h1>CONP</h1>
+              <p>Canadian Open Neuroscience Platform</p>
+            <a href="https://conp.ca/"><img src="images/CONP_logo.png"></a>
+                      <p><small>A national platform for open sharing of neuroscience research data.</small></p>
+                      <p><small><font color="E83812">Scroll down to explore some of CONP's initiatives.</font></small></p>
+                  </section>
+
+                  <!-- TOSI projects -->
+	    <section>
+	      <a href="https://www.mcgill.ca/neuro/open-science"><img src="images/Tosi_Dataset.png" width="76%"></a>
+	    </section>
+
+	    <!-- TOSI structure -->
+	    <section>
+	      <h2>Tannenbaum Open Science Initiative</h2>
+	      <img src="images/Tosi_Structure.png">
+	    </section>
+
+	    <!-- Cybersecurity -->
+	    <section>
+	      <img src="images/Cybersecurity.png">
+	    </section>
+              </section>
+
+          <!-- OHBM Column 6-->
+              <section>
+                  <section>
+              <h1>OHBM</h1>
+                      <p>Organization for Human Brain Mapping</p>
+            <a href="https://www.humanbrainmapping.org/i4a/pages/index.cfm?pageid=3267&pageid=1"><img src="images/OHBM_logo.png"></a>
+            <p><small>Dedicated to advancing our understanding of the human brain.</small></p>
+            <p><small><font color="E83812">Scroll down to explore some of OHBM's initiatives.</font></small></p>
+              </section>
+
+        <!-- Free as in beer-->
+	  <section data-transition="zoom">
+	    <a href="https://www.ohbmbrainmappingblog.com/blog/free-beer-and-free-speech-a-conversation-with-samir-das-and-pierre-bellec" target="_blank"><img src="images/Free_beer.png"></a>
 	  </section>
-	</section>
+                  </section>
 
-	<!--Open Science Technical Details -->
-	<section>
-	  
-	  <!-- Interoperability -->
+           <!-- Interoperability Column 7-->
```

Comment:
@christinerogers Not too familiar with most of these slides (lines 272-350, columns 7 and 8) so I just want to make sure that they are all relevant and necessary and up-to-date

Reply:
https://htmlpreview.github.io/?https://github.com/knoll-alex/samirdas.github.io/blob/ak-Open_Science/Open_Science_New.html#/7

https://htmlpreview.github.io/?https://github.com/knoll-alex/samirdas.github.io/blob/ak-Open_Science/Open_Science_New.html#/8